packages/react-image/src/Image.stories.tsx

```diff
@@ -31,6 +33,25 @@ const Stack = (props: React.PropsWithChildren<{ horizontal?: boolean }>) => {
   return <div {...rest} className={horizontal ? hStack : vStack} />;
 };
 
+export const Default = (props: ImageProps) => {
+  return <Image {...props} />;
+};
+Default.argTypes = {
+  alt: {
+    control: 'text',
+    defaultValue: 'Image placeholder',
```

Comment:
`alt` and `src` doesn't have any `@defaultValue` which is what we want, so in order to get proper playground we need to specify those manually (override-ing typescript generated controls)